modules/ROOT/pages/values-and-types/converting-data.adoc

```diff
@@ -0,0 +1,134 @@
+:description: This section provides information about how to convert data values using Cypher functions.
+[[converting-data-values]]
+= Converting data values 
+
+Cypher supports a number of functions to convert values to different data types.
+This section will provide an overview of those functions, as well examples of how to use them in practice. 
+
+== Functions for converting data values
+
+The following functions are available for converting data values:
+
+[options="header", cols="m,2a"]
+|===
+| Function | Description
+
+| toBoolean() | Converts a string, integer, or boolean value to a boolean value.
```

Comment:
I guess this is the functions actual description? bit strange it says it *converts* a boolean to a boolean when it leaves it as it (this applies to all function descriptions below)

Reply:
Yeah, I agree - and if we decide to change it, I will then change it in the functions pages as well (from which these descriptions have been taken)

Reply:
I made a trello for my team to look into the functions descriptions :)